easybuild/easyconfigs/k/Kalign/Kalign-2.0.4-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,28 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# updated by Denis Kristak (Inuits)
+
+easyblock = "ConfigureMake"
+
+name = "Kalign"
+version = "2.0.4"
```

Comment:
Any reason to introduce version 2 again? especially since you don't even use it as a dependency?
Multiple version of things used for deps are always a pain point for us, so unless there is a compelling need i would say to skip this completely.
If we truly do need this, then it must add the missing builddep on binutils.

Reply:
Oh, you're right, my fault to include it here.